modules/wri_program/wri_program.install

```diff
@@ -2,24 +2,33 @@
 
 /**
  * @file
- * Install and update hooks for wri_program.
+ * Update hooks for wri_program
  */
 
 /**
- * Add the 'Show More Link' field to the Program Center content type.
+ * Update the program display configs to remove blazy.
  */
 function wri_program_update_9301() {
 
+  \Drupal::service('distro_helper.updates')->installConfig('core.entity_view_display.node.program_center.card', 'wri_program', 'install', 'TRUE');
+
```

Comment:
These need to be re-arranged because the Show More Link update has run already. Make this one "9302" and the old one should stay "9301". Probably just a merge conflict.